app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/theme/Color.kt

```diff
@@ -7,5 +7,10 @@ val PurpleGrey40 = Color(0xFF625b71)
 val white = Color.White
 val transparent = Color.Transparent
 val black_70 = Color(0xB20C0404)
-val lightPurple = Color(0xFF8000FF)
-val darkPurple = Color(0xFF540BA1)
+val purple_100 = Color(0xFF8000FF)
+val purple_54 = Color(0xFF540BA1)
+
+val black_100 = Color(0xFF0C030C)
+val purple_36 = Color(0xFF36076B)
+val grey_50 = Color(0x80FFFFFF)
+val secondaryColor = Color(0xFFFF1F8A)
```

Comment:
Move this to say the correct color. `SecondaryColor` is being added in the theme